mac

```diff
@@ -124,14 +124,14 @@ brew unlink heroku
 brew link --force heroku
 
 fancy_echo "Configuring asdf version manager..."
-if [ ! -d "$HOME/.asdf" ]; then
-  git clone https://github.com/asdf-vm/asdf.git ~/.asdf --branch v0.3.0
-  append_to_shellrc "source $HOME/.asdf/asdf.sh" 1
-  . $HOME/.asdf/asdf.sh # For the current session as well
-else
-  append_to_shellrc "source $HOME/.asdf/asdf.sh" 1
-  . $HOME/.asdf/asdf.sh # For the current session as well
+if [ ! -d "/usr/local/opt/asdf" ]; then
+  brew install asdf
 fi
+append_to_shellrc "source /usr/local/opt/asdf/asdf.sh" 1
+append_to_shellrc "source /usr/local/etc/bash_completion.d/asdf.bash" 1
```

Comment:
I didn't think you had to manually load things in the bash_completion.d folder.

Reply:
looks like it's not needed, I'll remove this line